Makefile

```diff
@@ -61,40 +61,37 @@ INITC_IMAGE := kyvernopre
 initContainer: fmt vet
 	GOOS=$(GOOS) go build -o $(PWD)/$(INITC_PATH)/kyvernopre -ldflags=$(LD_FLAGS) $(PWD)/$(INITC_PATH)
 
-.PHONY: docker-build-initContainer docker-push-initContainer
+.PHONY: ko-build-initContainer
 
+# TODO(jason): Remove.
 docker-buildx-builder:
 	if ! docker buildx ls | grep -q kyverno; then\
 		docker buildx create --name kyverno --use;\
 	fi
 
-docker-publish-initContainer: docker-buildx-builder docker-build-initContainer docker-push-initContainer
+# TODO(jason): LDFLAGS
```

Comment:
I was going to ask about `LDFLAGS` ;)

Reply:
There will be a configuration file named `.ko.yaml`, for example, you can look at ko's default configuration [file](https://github.com/google/ko/blob/main/.ko.yaml#L7). Then, we pass these LDFLAGS to that file via the environment variable.

Reply:
Yep I meant to do that yesterday but just didn't have time before my son woke up from his nap 😆 

This should be done now, at least for `LD_FLAGS` -- I'll need to up my Makefile-fu to pass in the correct `LD_FLAGS_DEV`, then I can mark this as ready for review.